test/e2e/cross-pr/cross-pr.test.ts

```diff
@@ -158,13 +158,22 @@ test("PR from owner1/target:branchA to owner2/target:branchB while using mapping
       mockApi: [
         moctokit.rest.repos
           .get({
-            owner: "owner1",
```

Comment:
Ok this seems like it was a hack to get around the issue that we're fixing (to get into the other path of the getForkName method). Seems valid not to fail resolution of the repos.
I would add though a successful resolution of owner1/project4 - that's the one actually receiving a PR. Or do I miss sth?

Reply:
we already have a resolution of owner2/project4 in the block after this one.

In this cross-pr.test.ts, Owner1 has project1, project2, project3 and project4 while Owner 2 has only project3 and project4, that's why I mocked project1 and 2 to fail state and project 3 and 4 to ok state.

Reply:
I was talking about owner1/project4 , it should not affect how the test runs now, but it might "strengthen" the test context ... anyway, just a minor note really.

Reply:
@jstastny-cz I would go without is for now, as AFAIK we are not doing it for the other tests like full-downstream.test.ts, otherwise I think we would need to apply this everyone to be consistent (even not affecting the test result)

Reply:
@jstastny-cz if you are ok with the PR state, please approve it